spring-core/src/main/java/org/springframework/core/convert/support/GenericConversionService.java

```diff
@@ -166,6 +167,23 @@ public boolean canBypassConvert(@Nullable TypeDescriptor sourceType, TypeDescrip
 		return (converter == NO_OP_CONVERTER);
 	}
 
+	@SuppressWarnings("unchecked")
+	@Nullable
+	public <S, T> List<T> convertList(final @Nullable List<S> sourceList, final Class<T> targetClass) {
+
```

Comment:
I'm not sure if this line break is allowed by the code style. At least, I can't find any references to it. [Code Style](https://github.com/spring-projects/spring-framework/wiki/Code-Style)

Reply:
Removed final modifier.